src/main.ts

```diff
@@ -7,26 +7,23 @@ import router from './router'
 // Import stylesheet
 import 'vue-loading-overlay/dist/vue-loading.css'
 import 'bootstrap'
-import fetchPonyfill from 'fetch-ponyfill'
 
 Vue.config.productionTip = false
 
-// Work around for session key time out
-fetchPonyfill().fetch('/api/v2/i18n/lifelines-webshop/en').then((resp) => {
-  if (resp.status === 401) {
-    window.location.href = '/login'
-  }
-})
-
-Vue.use(i18n, {
-  lng: 'en',
-  fallbackLng: 'en',
-  namespace: ['lifelines-webshop', 'ui-form'],
-  callback () {
-    new Vue({
-      store,
-      router,
-      render: h => h(App)
-    }).$mount('#app')
-  }
-})
+store.dispatch('fetchContext').then(() => {
```

Comment:
magic 'fetchContext' string, i know where to find it , but who else is down line ?